### PR TITLE
dev-db/tarantool: add embed-luarocks flag

### PR DESCRIPTION
**Pull request by 0x501D**
Embed LuaRocks into tarantool binary if embed-luarocks is set.

### CODE REVIEW

File: dev-db/tarantool/tarantool-9999.ebuild [hunk 1]
```diff
@@ -41,7 +41,7 @@ HOMEPAGE="https://tarantool.org"
 IUSE="
 	+backtrace debug feedback-daemon gcov gprof +system-libcurl
 	+system-libyaml +system-zstd systemd test cpu_flags_x86_sse2
-	cpu_flags_x86_avx
+	cpu_flags_x86_avx embed-luarocks
```
Comment (Totktonada) [inline dev-db/tarantool/tarantool-9999.ebuild:44]: Nit: It seems, the flags are sorted alphabetically here (except `CPU_FLAGS_X86`, which are at the end).

File: dev-db/tarantool/metadata.xml [hunk 2]
```diff
@@ -19,6 +19,7 @@
 	<flag name="backtrace">Support for detail backtraces</flag>
 	<flag name="gcov">Build with coverage support</flag>
 	<flag name="gprof">Build with profiling support</flag>
+	<flag name="embed-luarocks">Embed LuaRocks into tarantool binary</flag>
```
Comment (Totktonada) [inline dev-db/tarantool/metadata.xml:22]: Nit: Same here, it seems the flags are sorted alphabetically.

**Review by Totktonada — APPROVED** [review]
Looks OK for me.

File: dev-db/tarantool/tarantool-9999.ebuild (hunk 1 above)
Comment (0x501D) [reply dev-db/tarantool/tarantool-9999.ebuild]: FIxed

File: dev-db/tarantool/metadata.xml (hunk 2 above)
Comment (0x501D) [reply dev-db/tarantool/metadata.xml]: Fixed

**Pull request merged**